store/orm/orm.go

```diff
@@ -195,7 +197,7 @@ func (orm *ORM) SaveJob(job *models.JobSpec) error {
 	for i := range job.Initiators {
 		job.Initiators[i].JobSpecID = job.ID
 	}
-	return orm.DB.Save(job).Error
+	return orm.DB.Create(job).Error
```

Comment:
Maybe we should go ahead and rename the method `CreateJob` to reflect this. And it's a great opportunity to use `:GoRename`

Reply:
That was fun ;)